operations/governance-remediation-process.md

```diff
@@ -0,0 +1,81 @@
+
+## Pre-escalation expectations
+
+Project members, stakeholders, and community members should first attempt to resolve governance issues through normal project discussion and consensus. The Remediation process exists for when normal, in-project resolution fails, either because of a difference of opinion, or because the project lacks the tools to make sufficient changes themselves.
+
+## Governance Remediation
+
+### Objectives:
+
+#### Establish Missing Governance 
+
+_Assist projects in developing governance structures to handle situations they are currently unequipped to manage._
+
+It's normal that as a project grows, it will encounter circumstances that weren't previously covered by the governance documentation, such that there is a lack of a clear path forward. This "missing governance" can usually be remediated amicably and internally within the project, by discussing proposed governance changes, and the maintainers agreeing to adopt changes. In these cases, missing governance is an opportunity for improvement. 
+
+#### Correct Non-Practiced Governance 
+
+_Ensure projects adhere to their documented governance._
+
+Non-practiced governance can either be someone willfully or inadvertently ignoring stated governance. Alternatively, occasions may arise where the defined governance _cannot_ be practiced as the project may be lacking people in a role to execute governance processes.
+
+When a governance conflict or unaccounted situation arises, and the project cannot reach a consensus to develop a path forward. The TOC will work with project stakeholders, TAG Contributor Strategy, the Code of Conduct Committee, and CNCF Staff to create a public remediation plan for the project.
+
+This plan could involve changes to the written governance, removal of specific project leaders, or whatever other measures are required to restore the project to a state where it is following its own governance rules, and that governance is in accord with CNCF principles. Should there be no way to restore the project to community operation, the TOC can consider archiving the project.
+
+In rare cases, governance failure may be accompanied by possible CoC violations by individual project leaders. Should this happen, the assigned team will engage the CNCF Code of Conduct Committee and the two teams will coordinate their evaluation.
+
+### Out of Scope:
+
+**Conduct Issues**: Handled by the CNCF Code of Conduct [processes](https://github.com/cncf/foundation/blob/main/code-of-conduct.md#reporting). This includes behavior by project leaders which is a violation of the CNCF CoC; such behavior should be reported to the CoCC and not handled through Governance Remediation.
+
```

Comment:
Now that I know that the TLP exists, we need to add that in here:

@TheFoxAtWork what are the correct links here?

```suggestion

**Leadership Issues**: Behavioral issues with individual project leaders may be addressed by contacting the TOC regarding violations of the Technical Leadership Principles.

```

Reply:
let me find out where this fell off. standby

Reply:
@joshgav to report potential violations and infringements of the Technical Leadership Principles, community members may report them to the private-toc@cncf.io mailing list. If the conduct is regarding a TOC member, community members may contact CNCF Staff, the TOC Chair, and Vice Chair.

CC @mrbobbytables

Reply:
Do we have a page for the TLP?

Reply:
Yes (sry I missed the early question) https://github.com/cncf/toc/blob/main/PRINCIPLES.md
About halfway down.